src/components/EditProfile/EditProfile.js

```diff
@@ -47,6 +55,23 @@ const EditProfile = props => {
     history.push('/')
   }
 
+  const handleDeleteAccount = () => {
+    const { confirm } = Modal
+    confirm({
+      title: 'Are you sure you want to delete your account item?',
```

Comment:
Suggestion: Update this to read 'Are you sure you want to delete your account?'.